docs/source/corev_hw_loop.rst

```diff
@@ -111,7 +111,7 @@ a matrix addition.
 
 
 At the beginning of the HWLoop, the registers %[i] and %[j] are 0.
-The innermost loop, from start0 to end0, adds to %[i] three times 1 and
+The innermost loop, from startO to endO, adds to %[i] three times 1 and
 it is executed 10x10 times. Whereas the outermost loop, from startO to endO,
```

Comment:
it's loop body is executed 10x10 times. Whereas the outermost loop, from startO to endO,